src/app/pages/SearchResultsPage/hooks.ts

```diff
@@ -112,31 +112,31 @@ export function useRuntimeAccountConditionally(
       query: {
         enabled:
           !!address &&
-          RouteUtils.getEnabledLayers().includes(Runtime.emerald) &&
+          RouteUtils.getEnabledLayersForNetwork(Network.mainnet).includes(Runtime.emerald) &&
           RouteUtils.getEnabledNetworks().includes(Network.mainnet),
```

Comment:
is `RouteUtils.getEnabledNetworks().includes(Network.mainnet)` still needed

Reply:
Currently, support for the mainnet can be easily disabled by editing `ENABLED_NETWORKS` in `route-utils.ts`.

Removing this condition would remove that ability.

Furthermore, I would like to preserve the uniformity of the four queries,  so that we can more easily replace them with a common, auto-generated form later on.

Reply:
OK